modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -28,17 +28,18 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStreamReader;
-import java.lang.ProcessBuilder.Redirect;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import org.aion.base.util.Hex;
 import org.aion.crypto.ECKey;
 import org.aion.crypto.ECKeyFac;
+import org.aion.db.utils.FileUtils;
```

Comment:
please copy the functionality you need instead of adding the dependency on the db module

Reply:
sure